dev/provider_packages/refactor_provider_packages.py

```diff
@@ -432,70 +432,6 @@ def amazon_package_filter(node: LN, capture: Capture, filename: Filename) -> boo
             .rename("airflow.providers.amazon.common.utils.email")
         )
 
-    def refactor_elasticsearch_package(self):
```

Comment:
What happened to this? Did you mean to remove it?

Reply:
Ah. Good point. I had really really bad problem with merge and I think this one is the one remaining.